corehq/motech/repeaters/models.py

```diff
@@ -1164,12 +1164,18 @@ def requeue(self):
         self.next_check = datetime.utcnow()
 
 
+# on_delete=DB_CASCADE denotes ON DELETE CASCADE in the database. The
+# constraints are configured in a migration. Note that Django signals
+# will not fire on records deleted via cascade.
+DB_CASCADE = models.DO_NOTHING
```

Comment:
Funny that Django doesn't have a built-in way to do this 🤷

Reply:
There's a [Django issue](https://code.djangoproject.com/ticket/21961) with a long discussion thread about it.